src/components/ClubCard.js

```diff
@@ -0,0 +1,149 @@
+    background-size: cover;
+    border-radius: ${props => props.theme.radius} ${props => props.theme.radius} 0 0;
+    display: block;
+    margin: 0;
+    padding-top: 66.6666%;
+    position: relative;
+    transition: 0.125s background-image ease-in, 0.125s opacity ease-in, 0.125s transform ease-in;
+    &:before {
+        background-image: url(/placeholder.svg);
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: contain;
+        content: "";
+        display: block;
+        height: 100%;
+        left: 0;
+        opacity: ${props => props.ready ? 0 : 0.25};
+        position: absolute;
+        right: 0;
+        top: 0;
+        width: 100%;
+    }
+    ${props => props.ready && css`
+        ${Inner}:hover & {
+            transform: scale(1.08);
+        }`
+    }
+`;
+
+const Actions = Flex.extend.attrs({
+    flexDirection: "row",
+    justify: "center",
+    wrap: true
+})``;
+
+const Action = Box.withComponent(Link).extend.attrs({
+    align: "center",
+    children: props => <FA icon={props.icon} />,
+    color: "primary",
+    p: 3,
+    target: "_blank"
+})`
+    flex-basis: 25%;
+    flex-grow: 1;
+    opacity: ${props => props.available ? 1 : 0.25};
+    transition: 0.125s background-color ease-in;
+`;
+
+class ClubCard extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { ready: false };
+    }
+
+    async componentWillMount() {
+        const { status } = await axios.get(`/school/${this.props.data.id}.jpg`);
+        if(status === 200) {
+            this.setState({ ready: true });
+        }
+    }
+
+    render() {
+        const { data, distance, showAllClubs, useImperialSystem } = this.props;
+        const { ready } = this.state;
+        return (
+            <Base>
+                <Inner>
+                    {!showAllClubs && <Distance>{geolib.convertUnit(useImperialSystem ? "mi" : "km", distance, 1)} {useImperialSystem ? "mi" : "km"} away</Distance>}
+                    <Box style={{ borderRadius: "4px 4px 0 0", overflow: "hidden" }}>
+                        <Photo src={`/school/${data.id}.jpg`} ready={ready} />
+                    </Box>
+                    <Flex p={3} justify="space-around" flexDirection="column" style={{ flex: 1 }}>
+                        <Heading.h4 style={{ textTransform: "capitalize" }}>{data.name}</Heading.h4>
+                        <Text pt={2}>{data.address}</Text>
+                    </Flex>
+                    <Actions>
+                        <Action icon="comment" />
+                        <Action href={`https://www.google.com/maps/place/${encodeURI(data.address)}`} icon="map" available />
+                        <Action icon={["fab", "slack-hash"]} />
+                        <Action icon="envelope" />
+                    </Actions>
+                </Inner>
+            </Base>
+        );
+    }
+}
+
+ClubCard.propTypes = {
+    data: PropTypes.object.isRequired
```

Comment:
this is like the minimum possible usage of prop-types :) would be great to describe the basic shape of your data, at least in a comment, to make contributing easier